File: Day37/matrixAddition.js
//Function for matrices addition
function getMatricesAddition(matrix1,matrix2){
    
    if (matrix1.length != matrix2.length  ||  matrix1[0].length != matrix2[0].length){
        console.log("Matrices cannot be added");
        return;
    }
    const rows = matrix1.length;
    const cols = matrix1[0].length;
    const result =[];

    for (let i = 0; i < rows; i++){
        const row =[];
        for(let j =0;j<cols;j++){
            row.push(matrix1[i][j] + matrix2[i][j]);
        }
        result.push(row);
    }
    return result;
}
//Let's start
matrix1 = [[1,2,3] , [4,5,6] , [7,8,9] ]
matrix2 = [ [10,20,30] , [40,50,60] , [70,80,90] ]

const result = getMatricesAddition(matrix1,matrix2);
//Displaying the result
console.log(result);

File: Day37/matrix_addition.c
#include<stdio.h>
//Let's start
void main(){
    int rows=3,cols=3;
    int matrix1[3][3]={
        {1,2,3},
        {4,5,6},
        {7,8,9}
    };
    int matrix2[3][3]={
        {10,20,30},
        {40,50,60},
        {70,80,90}
    };
    int result[rows][cols];
    for (int i = 0; i < rows; i++){
        for(int j =0;j<cols;j++){
            result[i][j] = matrix1[i][j] + matrix2[i][j];
        }
    }
    //Displaying the result
    printf("Resltant Matrix: \n");
    for (int i = 0; i < rows; i++){
        for(int j =0;j<cols;j++){
            printf("%d ",result[i][j]);
        }
        printf("\n");
    }
}
